internal/cli/alerts/settings/settings.go

```diff
@@ -157,9 +157,10 @@ func Builder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "settings",
 		Aliases: []string{"config"},
-		Short:   "Manage alerts configuration for your project.",
-		Long: `The settings command provides access to your alerts configurations.
-You can create, edit, and delete alert configurations.`,
+		Short:   "Manages alerts configuration for your project.",
+		Long: `The 'mongocli atlas alerts settings' command provides
+access to your alerts configurations. You can list, create, edit, and
+delete alert configurations.`,
```

Comment:
see my comment above, don't introduce new lines if not needed if the when using (`)
```suggestion
		Long: `The 'mongocli atlas alerts settings' command provides access to your alerts configurations. You can list, create, edit, and delete alert configurations.`,
```